frontend/src/modules/Dashboard/Components/AccountMenu.tsx

```diff
@@ -144,6 +144,11 @@ export const AccountMenu = ({
             Dashboard
           </MenuItem>
         )}
+        {showSettingsLink && (
+          <MenuItem component={Link} to="/dashboard/settings">
+            Settings
+          </MenuItem>
+        )}
```

Comment:
since this isn't in use yet i don't think you need to worry about putting it into the menu just yet (in case we deploy so we don't confuse the LSC staff).